### ISSUE TITLE
Tests for WCAG technique H74: Ensuring that opening and closing tags are used according to specification

### ISSUE THREAD

**Issue by jessebeach** [quailjs/quail#88]
This issue will most likely result in several tests. Please open more issues and link them as appropriate.

Most likely we will want to use a third-party library for this issue. This library will need to be MIT license compatible.
## Testing

From http://www.w3.org/TR/WCAG20-TECHS/H74.html
### Procedure
- [ ] Check that there are closing tags for all elements with required closing tags.
- [ ] Check that there are no closing tags for all elements where closing tags are forbidden.
- [ ] Check that opening and closing tags for all elements are correctly nested.
### Expected Results

Steps 1, 2, and 3 are true.

If this is a sufficient technique for a success criterion, failing this test procedure does not necessarily mean that the success criterion has not been satisfied in some other way, only that this technique has not been successfully implemented and can not be used to claim conformance.
